src/VisualStudio/Core/Def/Implementation/Remote/RemoteHostClientServiceFactory.RemoteHostClientService.cs

```diff
@@ -57,18 +54,14 @@ public void Enable()
                         return;
                     }
 
-                    if (!_workspace.Options.GetOption(RemoteHostOptions.RemoteHost))
+                    // We enable the remote host if either RemoteHostTest or RemoteHost are on.
+                    if (!_workspace.Options.GetOption(RemoteHostOptions.RemoteHostTest) &&
```

Comment:
@heejaechang I don't quite get the purpose of "RemoteHostTest".  If tests want to enable REmoteHost, why don't they just set "RemoteHost" to true?